en/docs/install-and-setup/setup/setup-overview.md

```diff
@@ -0,0 +1,41 @@
+# Setup Overview
+
+Setting up involves doing the required configurations for the API Manager and its components before running them in the production environment. The following are some common set up tasks that you are required to do separately for each component.
+
+- Configuring databases
+- Configuring user stores
+- Configuring security
+- Setting up observability solutions
+- Configuring transport
+- Performance Tuning
+
+The above activities need to be carried out separately for each runtime. The procedures to execute them are similar, but there can be slight differences between one component to another.
+
+In addition, the component-specific setup tasks are as follows:
+
+- API Manager
+
+    - Configuring key managers
```

Comment:
Can these be links? Also it'll be nice if we can have a small description of each.

Reply:
If they are links, there will need to be 3 links per topic. The content will need to be duplicated for the 3 components due to  differences.

Reply:
I feel links are useful because they will give the user some direction and help with navigation. This does not need to be an exhaustive list of links of course, just some important ones to take people to the places they might be interested in.

Reply:
Added links